language_names/manual.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+import textwrap
+import csv
+
+
+def insert_values(c, filename):
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS {file} (
+            id INTEGER PRIMARY KEY,
+            lg TEXT,
+            inLg TEXT,
+            name TEXT,
+            UNIQUE(lg, inLg) ON CONFLICT REPLACE);
+        """.format(file=filename)))
+    with open(filename + '.tsv', 'r') as f:
+        reader = csv.reader(f, delimiter='\t')
```

Comment:
Using a `DictReader` will allow avoiding the weird `next(...)` below.

Reply:
yeah that would be great